Libraries/Components/WebView/WebView.android.js

```diff
@@ -127,13 +134,25 @@ var WebView = React.createClass({
       domStorageEnabled = this.props.domStorageEnabledAndroid;
     }
 
+    var htmlProp, urlProp, sourceProp;
+    if(this.props.html) {
+      console.warn('html is deprecated. Use source instead.');
+      htmlProp = this.props.html;
+    } else if(this.props.url) {
+      console.warn('url is deprecated. Use source instead.');
```

Comment:
@nicklockwood That'll be awesome. Though I'm not aware of any such functionality in React. Maybe we could do this with `propTypes`?